var/spack/repos/builtin/packages/axl/package.py

```diff
@@ -45,7 +45,8 @@ class Axl(CMakePackage):
     tags = ['ecp']
 
     version('master', branch='master')
-    version('0.1.0', '1ff16c046c3a080c252e0bf4251b83bc')
```

Comment:
Can we keep the previous release?

Reply:
No, it was unusable (major fixes include adding `extern "C"` protection, which is required for VELOC).